common/lib/xmodule/xmodule/js/src/video/03_video_player.js

```diff
@@ -109,16 +109,7 @@ function(HTML5Video, Resizer) {
         // starts playing. Just after that configurations can be applied.
         state.videoPlayer.ready = _.once(function() {
             if (!state.isFlashMode() && state.speed != '1.0') {
-                // Work around a bug in the Youtube API that causes videos to
-                // play at normal speed rather than at the configured speed in
-                // Safari.  Setting the playback rate to 1.0 *after* playing
-                // started and then to the actual value tricks the player into
-                // picking up the speed setting.
-                if (state.browserIsSafari && state.isYoutubeType()) {
```

Comment:
@karandikar Since we're not using these variables anymore `browserIsFirefox` / `browserIsChrome` / `browserIsSafari`
Can we remove them from the code, also we need to remove them from spec files too.